src/utils/wallet.spec.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { expect } from "chai";
+import { signMessage, verifyMessage } from "./wallet";
+
+const mockConfigForWallet1: any = {
+    get: () =>
+        "afdfd9c3d2095ef696594f6cedcae59e72dcd697e2a7521b1578140422a4f890"
```

Comment:
Could be done as `const TEST_PRIV_KEY_ONE = etc...` and then passed in here but not a big deal